src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -486,27 +487,46 @@ def add_digital_object(self, parent_archival_object, dashboard_uuid, title="", i
             "notes": [{
                 "jsonmodel_type": "note_digital_object",
                 "type": "originalsloc",
-                "content": [dashboard_uuid]
+                "content": [dashboard_uuid],
+                "publish": True,
             }],
             "restrictions": restricted,
             "subjects": parent_record['subjects'],
             "linked_agents": parent_record['linked_agents'],
         }
 
+        i = 0
```

Comment:
That i variable is not actually needed.  In an rough version of the code, that I did not commit, the count was being used, but I realized it wasn't necessary, but forgot to remove it.

I have removed it in a new commit now